agent/lib/kontena/service_pods/creator.rb

```diff
@@ -191,7 +189,7 @@ def image_outdated?(image_name, service_container)
       # @return [Boolean]
       def recreate_service_container?(service_container)
         state = service_container.state
-        service_container.restart_policy['Name'] == 'always' &&
+        service_container.restart_policy['Name'] == 'unless-stopped' &&
```

Comment:
What Docker version started to support `unless-stopped`. Does this have affect on the minimum Docker version that Kontena supports?